frontend/packages/console-shared/src/utils/storage-utils.ts

```diff
@@ -7,13 +7,13 @@ export const cephStorageProvisioners = [
   'rbd.csi.ceph.com',
 ];
 
+const objectStorageProvisioners = ['noobaa.io/obc', 'openshift-storage-ceph.rook.io/bucket'];
+
 // To check if the provisioner is OCS based
 export const isCephProvisioner = (scProvisioner: string): boolean => {
   return cephStorageProvisioners.some((provisioner: string) =>
     _.endsWith(scProvisioner, provisioner),
   );
 };
 
-// To check that the storage class isn't noobaa based
-export const filterScOnProvisioner = (sc: StorageClass, provisioner: string = '') =>
-  sc.provisioner.includes(provisioner);
+export const isObjectSC = (sc: StorageClass) => objectStorageProvisioners.includes(sc.provisioner);
```

Comment:
```suggestion
export const isObjectSC = (sc: StorageClass): boolean => {
  return objectStorageProvisioners.some((provisioner: string) =>
    sc.provisioner.includes(provisioner)
  );
};

```

Reply:
This is not required if we modify the SC name to `openshift-storage.noobaa.io/obc`